app/models/course/assessment.rb

```diff
@@ -40,6 +40,9 @@ class Course::Assessment < ActiveRecord::Base
   has_many :programming_questions,
            through: :questions, inverse_through: :question, source: :actable,
            source_type: Course::Assessment::Question::Programming.name
+  has_many :scribing_questions,
```

Comment:
Can you shift this and the abilities into a separate commit, then controllers + views go into 1 commit?